portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,61 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.sps.data.DataStats;
+import com.google.gson.Gson;
+import java.util.Date;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+   private final Date dateCreated = new Date();
+   public ArrayList<String> jsonData = new ArrayList<String>();
+
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.setContentType("application/json");
+    String json = new Gson().toJson(jsonData);
+    response.getWriter().println(json);
+  }
+
+   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+       String[] userInputs = getUserInputs(request);
+       String userName = userInputs[0];
+       String userComment = userInputs[1];
+       DataStats dataStats = new DataStats(dateCreated, userName, userComment);
+       String json = convertToJson(dataStats);
+       jsonData.add(json);
+       response.setContentType("application/json;");
+       response.getWriter().println(jsonData);
+   }
+
+    private String[] getUserInputs(HttpServletRequest request) {
+        String userNameString = request.getParameter("name-choice");
+        String userCommentString = request.getParameter("comment-choice");
+        String[] inputs = { userNameString, userCommentString};
+        return inputs;
+    }
+
+ private String convertToJson(DataStats dataStats) {
+    String json = "{";
+    json += "\"dataCreated\": ";
+    json += "\"" + dataStats.getTime() + "\"";
+    json += ", ";
+    json += "\"userName\": ";
+    json += "\"" + dataStats.getUserName()  + "\"";
+    json += ", ";
+    json += "\"userComment\": ";
+    json += dataStats.getComment();
+    json += "}";
+    return json;
   }
+
+   private String convertToJsonUsingGson(DataStats dataStats) {
```

Comment:
this method is not used. do you meant to replace the method above with this one?